lib/internal/util.js

```diff
@@ -213,6 +214,14 @@ function getConstructorOf(obj) {
   return null;
 }
 
+function getErrorName(code) {
+  if (errmap.has(code)) {
+    return errmap.get(code)[0];
+  } else {
+    return `Unknown system error ${code}`;
+  }
```

Comment:
Rather than the separate `has()` and `get()` calls, perhaps just

```js
function getErrorName(code) {
  const entry = errmap.get(code);
  return entry ? entry[0] : `Unknown system error ${code}`;
}
```